### ISSUE TITLE
Support multiple image sources for ryuk

### ISSUE THREAD

**Issue by jkroepke** [testcontainers/testcontainers-java#2813]
Looking at https://github.com/testcontainers/testcontainers-java/blob/ecdbc11c2d637d3dec50b32655958a9a8ef1e807/docs/features/configuration.md#customizing-ryuk-resource-reaper

I saw it's possible to define a custom image source for ryuk:
```
ryuk.container.image = quay.io/testcontainers/ryuk:0.2.3
```

It would be great if its possible to define multiple images like:

```
ryuk.container.image = quay.io/testcontainers/ryuk:0.3.0 testcontainersofficial/ryuk:0.3.0
```

Why? If case the first location is offline use the second one.

If quay.io is done for some reasons, a lot of users will expect error in the own projects integrations sources.

**Comment by bsideup** [comment]
Hi @jkroepke,

This issue is a duplicate of #2763. See the discussion there.

tl;dr: we're migrating to Docker Hub, off Quay.io completely, and IMO it does not make sense to even have it as a fallback.
